scripts/storage-emulator-integration/tests.ts

```diff
@@ -263,9 +263,51 @@ describe("Storage emulator", () => {
           });
         });
 
-        // TODO(abehaskins): This test is temporarily disabled due to a credentials issue
-        it.skip("should handle large (resumable) uploads", async () => {
-          await testBucket.upload(largeFilePath, { resumable: true });
+        it.only("should handle large (resumable) uploads", async () => {
+          const fileName = "test_upload.jpg";
+          const uploadURL = await supertest(STORAGE_EMULATOR_HOST)
+            .post(`/upload/storage/v1/b/${storageBucket}/o?name=${fileName}&uploadType=resumable`)
+            .send({})
```

Comment:
Do you need this `send` then?

Reply:
Yup, without it we throw a `400 "Bad Request"`